packages/EDA/src/package.ts

```diff
@@ -542,3 +546,213 @@ export function anova(table: DG.DataFrame, factor: DG.Column, feature: DG.Column
 export function kNNImputation() {
   runKNNImputer();
 }
+
+//name: Linear regression
+//description: Linear regression demo
+//input: dataframe table
+//input: column_list features {type: numerical}
+//input: column target {type: numerical}
+//input: bool plot = true {caption: plot}
+export async function linearRegression(table: DG.DataFrame, features: DG.ColumnList, target: DG.Column, plot: boolean): Promise<void> {
```

Comment:
Not sure what this function is doing - I mean, how is it making Datagrok better?
Does it modify the original table?

Reply:
It's a function for demo performance linear regression. REMOVED!